ui/src/app/pages/analysis/runtime-panel.tsx

```diff
@@ -883,20 +1088,43 @@ export const RuntimePanel = fp.flow(
   if (currentRuntime === undefined) {
     return <Spinner style={{width: '100%', marginTop: '5rem'}}/>;
   }
-
+  const createStandardComputeRuntimeRequest = (runtime: RuntimeConfig) => {
+    // The logic here is tricky to be compatible
+    // Use gceConfig when (PD feature is not enabled) OR (increase PD size or update machineType of an existing runtime with PD attached)
+    // OR (update an existing runtime with no PD attached).
+    // TODO(RW-): 'Update an existing runtime with no PD attached' will be cleaned up
```

Comment:
Please add the ticket number

Reply:
I didn't create that ticket afterwards. It's difficult to separate one single operation before this entire interaction done.

Reply:
CH's suggestion is to put a real ticket number([create one in Jira](https://precisionmedicineinitiative.atlassian.net/secure/RapidBoard.jspa?rapidView=30&projectKey=RW))
and put that in 1095. More context: I added line 1095 when pairing with Yintong. There should be a follow up TODO item to clean things up. TODO items should be tracked/labeled with a ticket number. (See [go/todo-style](go/todo-style))